### PR TITLE
FOCUS v1.0 - preview payload

### PR DESCRIPTION
**Pull request by udam-f2**

### CODE REVIEW

File: specification/columns/servicecategory.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Service Category
 
-The Service Category is the highest-level classification of a service based on the core function of the service. Each service should have one and only one category that best aligns to its primary purpose. The Service Category is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.
+The Service Category is the highest-level classification of a [*service*](#glossary:service) based on the core function of the *service*. Each *service* should have one and only one category that best aligns to its primary purpose. The Service Category is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.
```
Comment (yprikhodko) [inline specification/columns/servicecategory.md:3]: Can we include this PR as well - https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/251?

Comment (udam-f2) [reply specification/columns/servicecategory.md:3]: I think this is resolved. Closing. 

@yprikhodko responded to you on slack. Maintainers decided #251 would be better added post preview release (before v1.0) after feedback. Appreciate you bringing it up.

**Review by mike-finopsorg — APPROVED** [review]

File: specification/attributes/column_naming_convention.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
 # Column Naming Convention
 
-Column IDs provided in cost data following a consistent naming convention reduces friction for FinOps practitioners
-that consume the data for analysis, reporting, and other use cases.
+Column IDs provided in cost data following a consistent naming convention reduces friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
```
Comment (thecloudman) [inline specification/attributes/column_naming_convention.md:3]: > `Column IDs provided in cost data following a consistent naming convention **reduce** friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.`
```suggestion
Column IDs provided in cost data following a consistent naming convention reduce friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
```

**Review by thecloudman — CHANGES_REQUESTED** [review]

File: specification/attributes/column_naming_convention.md (hunk 2 above)
Comment (mike-finopsorg) [reply specification/attributes/column_naming_convention.md:3]:
```suggestion
Column IDs provided in cost data that follow a consistent naming convention reduce friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
```

Comment (thecloudman) [reply specification/attributes/column_naming_convention.md:3]:
```suggestion
Column IDs provided in cost data which follow a consistent naming convention reducing friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
```

**Review by thecloudman — CHANGES_REQUESTED** [review]
I like the addition you added @flanakin and i forgot to add "ing" so updated with what i actually meant

**Review by thecloudman — APPROVED** [review]

**Comment by marc-perreaut** [comment]
Maybe "indictating" could be replaced with "indicating" in numeric_format.md, line 23.

https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/attributes/numeric_format.md?plain=1#L23

**Comment by AndersonOliveira2022** [comment]
On Overview.md, the word **providers** is in italic, but missing the link.

**Pull request merged**